storybook/stories/Examples/AreaChart.stories.tsx

```diff
@@ -407,3 +407,81 @@ export const AreaChartFillByValue = {
     );
   },
 };
+
+export const StackAreaWithDiscontinuousValues = {
+  render: () => {
+    const data = [
+      {
+        x: 0,
+        b: 1,
+        c: 1,
+      },
+      {
+        x: 1,
+        a: 1,
+        b: 1,
+        c: 1,
+      },
+      {
+        x: 2,
+        a: 1,
+        b: 1,
+        c: 1,
+      },
+      {
+        x: 3,
+        b: 1,
+        c: 1,
+      },
+      {
+        x: 4,
+        b: 1,
+        c: 1,
+      },
+    ];
+
+    const colors = {
+      a: 'red',
+      b: 'blue',
+      c: 'green',
+    } as const;
+
+    return (
+      <AreaChart data={data} height={200} width={300}>
+        {(['a', 'b', 'c'] as const).map(k => (
+          <Area key={k} dataKey={k} stroke={colors[k]} fill={colors[k]} stackId={1} connectNulls />
+        ))}
+      </AreaChart>
+    );
+  },
+};
+
+export const StackAreaWithDiscontinuousValuesTwo = {
+  render: () => {
+    const data = [
+      { month: '2015.01', a: 4000, b: 2400, c: 2400 },
+      { month: '2015.02', a: 3000, b: 1398, c: 2210 },
+      { month: '2015.03', a: 2000, b: 9800, c: 2290 },
+      { month: '2015.04', a: 2780, b: 3908, c: 2000 },
+      { month: '2015.05', a: 1890, b: 4800, c: 2181 },
+      { month: '2015.06', a: 1000 },
+      { month: '2015.07', a: 2390, b: 3800, c: 2500 },
+      { month: '2015.08', a: 3490, b: 4300, c: 2100 },
+      { month: '2015.09' },
+    ];
+
+    const toPercent = (decimal: number, fixed = 0) => {
+      return `${(decimal * 100).toFixed(fixed)}%`;
+    };
+
+    return (
+      <AreaChart width={600} height={400} data={data} margin={{ top: 10, right: 30, left: 0, bottom: 0 }}>
```

Comment:
the y axis on the story gets cut off so I think we need to increase left margin here